src/basicstruct.py

```diff
@@ -24,6 +25,20 @@ def __init__(self, *args, **kwargs):
             if not hasattr(self, key):
                 setattr(self, key, None)
 
+    def to_dict(self, copy=False):
+        """Convert the struct to a dictionary.
+
+        If `copy == True`, returns a `copy.deepcopy` of the values.
```

Comment:
Just say "returns a deep copy of the values". The specific method is an implementation detail.